src/mcp/tools/crashlytics/events.ts

```diff
@@ -1,21 +1,117 @@
 import { z } from "zod";
 import { tool } from "../../tool";
+import { dump, DumpOptions } from "js-yaml";
 import { batchGetEvents, listEvents } from "../../../crashlytics/events";
 import {
   BatchGetEventsResponse,
+  Breadcrumb,
   ErrorType,
   Event,
+  Exception,
+  Frame,
   ListEventsResponse,
+  Log,
+  Thread,
+  Error,
 } from "../../../crashlytics/types";
 import { ApplicationIdSchema, EventFilterSchema } from "../../../crashlytics/filters";
-import { mcpError, toContent } from "../../util";
+import { mcpError } from "../../util";
 
-function pruneThreads(sample: Event): Event {
-  if (sample.issue?.errorType === ErrorType.FATAL || sample.issue?.errorType === ErrorType.ANR) {
-    // Remove irrelevant threads from the response to reduce token usage
-    sample.threads = sample.threads?.filter((t) => t.crashed || t.blamed);
+const DUMP_OPTIONS: DumpOptions = { lineWidth: 200 };
+
+function formatFrames(origFrames: Frame[], maxFrames = 20): string[] {
```

Comment:
What happens if the blame frame is beyond the maxFrames?

Reply:
The blame frame is actually separated out into its own top-level section, so it's always accessible.

Reply:
Ahh, perfect!